ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -105,6 +106,14 @@ public async Task<Notification> GetNotificationWithTestsAsync(int notificationId
                 .FirstOrDefaultAsync(n => n.NotificationId == notificationId);
         }
 
+        public async Task<Notification> GetNotificationWithSocialContextVenuesAsync(int notificationId)
+        {
+            return await GetBaseNotificationsIQueryable()
```

Comment:
will need an update to `GetBannerReadyNotificationsIQueryable` from the performance-tackling change.

Reply:
Ok will check this when merging :)